File: app/src/main/java/com/iogarage/ke/pennywise/BaseActivity.java
package com.iogarage.ke.pennywise;


import com.tapadoo.alerter.Alerter;

import io.multimoon.colorful.CAppCompatActivity;

public class BaseActivity extends CAppCompatActivity {


    void showErrorAlert(String title, String message) {
        Alerter.create(this)
                .setTitle(title)
                .setText(message)
                .setBackgroundColorRes(R.color.coral) // or setBackgroundColorInt(Color.CYAN)
                .show();
    }

    void showInfoAlert(String title, String message) {
        Alerter.create(this)
                .setTitle(title)
                .setText(message)
                .setBackgroundColorRes(R.color.md_blue_500) // or setBackgroundColorInt(Color.CYAN)
                .show();
    }
}

File: app/src/main/java/com/iogarage/ke/pennywise/views/BaseActivity.kt
package com.iogarage.ke.pennywise.views

import androidx.appcompat.app.AppCompatActivity
import com.iogarage.ke.pennywise.R
import com.tapadoo.alerter.Alerter
import dagger.hilt.android.AndroidEntryPoint

@AndroidEntryPoint
open class BaseActivity : AppCompatActivity() {
    fun showErrorAlert(title: String?, message: String?) {
        Alerter.create(this)
            .setTitle(title!!)
            .setText(message!!)
            .setBackgroundColorRes(R.color.coral) // or setBackgroundColorInt(Color.CYAN)
            .show()
    }

    fun showInfoAlert(title: String?, message: String?) {
        Alerter.create(this)
            .setTitle(title!!)
            .setText(message!!)
            .setBackgroundColorRes(R.color.md_blue_500) // or setBackgroundColorInt(Color.CYAN)
            .show()
    }
}